services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/TableSchema.java

```diff
@@ -240,8 +241,24 @@ default T mapToItem(Map<String, AttributeValue> attributeMap, boolean preserveEm
      * @param ignoreNulls If set to true; any null values in the Java object will not be added to the output map.
      *                    If set to false; null values in the Java object will be added as {@link AttributeValue} of
      *                    type 'nul' to the output map.
+     * @param requestConfiguration denotes the level of nesting, i.e. SHALLOW or NESTED, contained in the request params based
+     * which nested update query parameters would be dealt with
```

Comment:
Maybe `configuration options for mapping an object to a map with attribute values`?